src/main/navigation/navigation_fw_launch.c

```diff
@@ -316,10 +316,11 @@ static fixedWingLaunchEvent_t fwLaunchState_FW_LAUNCH_STATE_WAIT_DETECTION(timeU
 
     const bool isBungeeLaunched = isForwardAccelerationHigh && isAircraftAlmostLevel;
     const bool isSwingLaunched = (swingVelocity > navConfig()->fw.launch_velocity_thresh) && (imuMeasuredAccelBF.x > 0);
+    const bool isForwardLaunched = (gpsSol.groundSpeed > navConfig()->fw.launch_velocity_thresh) && (imuMeasuredAccelBF.x > 0);
```

Comment:
Note that launch mode is also useful for planes _without_ GPS. This should explicitly check for GPS presence and GPS fix.

Reply:
That's a good point. I figured if you don't have a gps connected then this bool will never be true anyway so no harm done. But I can add an explicit check.

Reply:
I replaced `gpsSol.groundSpeed > navConfig()->fw.launch_velocity_thresh` with `isGPSHeadingValid()` which does all the necessary checks and uses the same 3m/s threshold as the default `launch_velocity_thresh`.